test_main.py

```diff
@@ -34,7 +38,7 @@ def test_get_books(self, test_db):
         book2 = create_book(test_db, BookIn(**TEST_BOOKS[1]))
 
         books = get_books(test_db)
```

Comment:
```suggestion
```suggestion
        assert len(books) >= 2  # Adjusted expected count
```
```